R/derive_summary_records.R

```diff
@@ -196,12 +196,39 @@ derive_summary_records <- function(dataset,
     subset_ds <- dataset
   }
```

Comment:
This part (lines 189-197) can be removed as the filtering is done in `get_summary_records()`.

Reply:
updated the code. Please check.

Reply:
The filtering code is still there.